source/includes/_changelog.md.erb

```diff
@@ -4,14 +4,19 @@
 
 ```http
 GET https://api.teamtailor.com/v1/jobs HTTP/1.1
-X-Api-Version : 20161108
+X-Api-Version : 20201113
 ```
 
 ```shell
 curl -X "GET" "https://api.teamtailor.com/v1/jobs" \
-     -H "X-Api-Version: 20161108"
+     -H "X-Api-Version: 20201113"
 ```
 
+### 2020-11-16 *Breaking update*
```

Comment:
But it's not ”breaking" if you keep using your old version right?

Maybe say "New version" instead or something?